.ci/generate_package.py

```diff
@@ -95,7 +95,8 @@ def create_upload_parser():
 def main():
     args = create_upload_parser().parse_args()
 
-    os.chdir(os.path.join(os.path.dirname(__file__), '..'))
+    current_dir = os.path.dirname(__file__)
```

Comment:
this shouldnt need changing.

other than the bears directory, the script shouldnt rely on anything else being in some specific location.